workflows/src/main/kotlin/com/r3/corda/lib/ci/SyncKeyMappingFlow.kt

```diff
@@ -14,14 +14,21 @@ import net.corda.core.utilities.unwrap
 import java.security.PublicKey
 
 /**
- * This flow allows a node to share the [PublicKey] to [Party] mapping data of parties unknown parties present in a given
- * transaction. The initiating node sends a list of confidential identities to the counter-party who attempts to resolve
- * them. Parties that cannot be resolved are returned to the initiating node.
+ * This flow allows a node to share the [PublicKey] to [Party] mapping data of parties unknown to the counter-party. The
+ * two constructors allow the the node to provide a transaction in which the confidential identities unknown to the counter-party
```

Comment:
typo: the the node

Reply:
Still unresolved. Please don't mark comments as resolved when you didn't address them